packages/ui/src/components/Button/Button.stories.tsx

```diff
@@ -64,3 +64,33 @@ LoadingButton.args = {
     icon: Icon,
     isLoading: true,
 };
+
+export const Badge = ButtonTemplateWithChild.bind({});
```

Comment:
Let's also rename the stories to `Rounded` so that it's clear how its connected to the prop.
I believe the naming in the Figma designs should also be updated.

Reply:
By the way, shouldn't these stories be connected to Figma designs? 😉

Reply:
Updated